[DM-3316]Devicemngt App - All devices - filter not available for smaller screensizes (#1917)
SoftwareAG/c8y-docs: content/change-logs/device-management/cumulocity-10-20-0-0-information-about-data-grid-filter-not-available-in-mobile-screens-was-added-to-codex-documentation.md

---
date: ""

> title: Data Grid filter not available in mobile screens info added to Codex documentation.

Reviewer: ```suggestion
title: Info added to Codex documentation on data grid filter not available in mobile screens
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:

>   - value: tc-QHwMfWtBk7

Reviewer 2: ```suggestion
  - value: tc-pjJiURv9Y
```

    label: ui-c8y
ticket: DM-3316
version: 1020.0.0
---

> Information about Data Grid filter (and other functionalities) not being available in mobile screens was added to Codex documentation.

Reviewer: ```suggestion
The information that the data grid filter functionality as well as other functionalities are not available in mobile screens has been added to the Codex documentation.
```

> The information that the data grid filter functionality as well as other functionalities are not available in mobile screens has been added to the Codex documentation.

Reviewer: ```suggestion
The information that the data grid filter as well as other functionalities are not available in mobile screens has been added to the Codex documentation.
```
